### PR TITLE
chore: improve functionality of release notes

### PR DESCRIPTION
**Pull request by rudouglas**
I've used a Python script to pull more detailed data from GitHub, like usernames, and to allow us to customize more easily going forward

### CODE REVIEW

**Review by nr-kkenney — COMMENTED** [review]
id be in favor of having this live in a python file rather than in the body of the workflow. feel free to grab another opinion on this, since this is just my personal opinion 🙈

**Comment by rudouglas** [comment]
i'll take all opinions @nr-kkenney personal, professional or other wise 😂 This would probably be better ye, where would be the best place for this to live, just in the `github` folder?

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -18,34 +18,122 @@ jobs:
...
           
-      
+      - name: Install Python dependencies
+        id: install-python
+        run: |
+          python -m pip install --upgrade pip
+          pip install PyGithub
+          pip install invoke
+          
+      - name: Create Updates
```
Comment (nr-kkenney) [inline .github/workflows/release.yml:75]: haha. i should have commented on this so we could get a thread. i think moving this under .github is 👍  ive done .github/scripts for the demotron repo. im pretty open as to the exact path, but as long as its under .github, sounds good to me.

Comment (rudouglas) [reply .github/workflows/release.yml:75]: ha i don't know why you can't just make a thread without commenting 🙈  excellent ye that sounds much better, keep it all neat and tidy, thanks

**Pull request closed without merging**